lisp/bazel-build.el

```diff
@@ -24,27 +24,42 @@
 
 ;;; Code:
 
+(defgroup bazel-build 'nil
+  "Package for editing, building, and running code using Bazel."
+  :link '(url-link "https://github.com/bazelbuild/emacs-bazel-mode")
+  :group 'languages)
+
+;;;###autoload (put 'bazel-build-bazel-command 'safe-local-variable 'file-executable-p)
```

Comment:
I think this shouldn't be marked as safe. With this being "safe", people can put arbitrary commands into the file-local variable list, I don't think that's what we want.

Reply:
I added this in response to @sellout’s comment on the other PR (https://github.com/bazelbuild/emacs-bazel-mode/pull/16#discussion_r388960113). I agree with your comment though.

Would making this a buffer-local variable make more sense then? I’ve seen environments that could benefit, but this use case might be too niche.

Reply:
Another way to look at it might be to let people mark this variable as safe in their `init-early.el` configuration if they so choose? That would let us stay on the safer side by default.

Reply:
I don't think this should be buffer-local. The Bazel command is typically a system-wide settings (e.g. users could install Bazel outside of their PATH), not a file-specific one.

I'm pretty convinced that this variable should never be marked safe. Doing so looks like a significant security risk, especially if we consider something like a Flymake backend later. In fact, I think it should be marked as risky (which Emacs does automatically for variables ending in "-command", but we might as well be explicit here).

Reply:
Sold. Marked as risky!